### PR TITLE
Pluggable advertisement example and tutorial - iOS

### PR DESCRIPTION
**Pull request by pablorueda**
Pluggable advertisement example and tutorial - iOS

### CODE REVIEW

File: Advertisement/iOS/README.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+``` swift
+    public func createAdPresenter(adView:ZPAdViewProtocol, parentVC:UIViewController) -> ZPAdPresenterProtocol {
+        return MyAdvertisementPresenter(adView: adView, parentVC: parentVC)
+    }
+```
+
+Here we are passing as parameters the view protocol that Applicaster SDK will implement and that our presenter is going to use to call some methods, and the parent view controller that we will need to use to present the interstitial or in case we need access to some of their parameters.
+
+After we need to implement a method that is used by our analytics system to know the name of the plugin
```
Comment (lomudi) [inline Advertisement/iOS/README.md:79]: @PRuedaApplicaster can you explain me a little bit about this? I'm not familiar with that.

**Review by lomudi — COMMENTED** [review]
@PRuedaApplicaster looks good and working. I only had a small question.

File: Advertisement/iOS/README.md (hunk 1 above)
Comment (pablorueda) [reply Advertisement/iOS/README.md:79]: The analytics point?
Basically the analytics plugins are passing the name of the ad provider as one of the parameters when tracking an event (currently we are using just 'DFP' as it is the only provider we use).

Comment (pablorueda) [reply Advertisement/iOS/README.md:79]: Here are the docs to know more about it:
https://docs.google.com/spreadsheets/d/1dn23ciC4Q14DQPXhIv4Kbv9woPhO_tTCayvWXVVRS-k/edit?usp=sharing
(in the second tab you can see we need the ad provider name)

**Review by lomudi — APPROVED** [review]
💯

**Pull request merged**